### PR TITLE
Add NoLoginRequired in Readme file

### PR DESCRIPTION
**Pull request by sauravrao637**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [x] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- record videos
- create clones
- take over the world

#### Fixes the following issue(s)

- Fixes #

#### Relies on the following changes

- Need to make appropriate changes in other translations of README

#### APK testing 

On the website the APK can be found by going to the "Checks" tab below the title and then on "artifacts" on the right.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Stypox** [comment]
This should be added to all readmes

**Comment by sauravrao637** [comment]
@Stypox  let's add in one and then can open an issue for adding translations in other

**Comment by Stypox** [comment]
I think this can be translated with an online translator and updated in all readmes, then if a traslator thinks he has a better translation he can always change it. Otherwise it would be difficult to gather all translators

**Comment by sauravrao637** [comment]
@Stypox  done :)

**Review by mhmdanas — DISMISSED** [review]

File: README.so.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ NewPipe ma isticmalo nidaamka wada shaqaynta Google, ama API-ga YouTube. Kaliya
 ### Waxqabadka
 
 * Raadi muuqaalo
+* Login Looma Baahan
```
Comment (nadiration) [inline README.so.md:48]: @sauravrao637 This translation is wro

**Comment by nadiration** [comment]
@sauravrao637 This translation [Somali] is wrong. You should never use Google translate. If you don't know a language leave it be.

**Comment by nadiration** [comment]
@Stypox That's is not a good idea. Online translations (Google Translate etc) are not 'acurate', they are out of context and confusing plus it's not the same with all languages. Some languages are a mess. Can't Readmes be on Weblate?

> I think this can be translated with an online translator and updated in all readmes, then if a traslator thinks he has a better translation he can always change it. Otherwise it would be difficult to gather all translators

**Comment by mhmdanas** [comment]
I agree with @nadiration. I think it would be best in general to just edit the languages that one knows in a PR and leave the rest for other PRs by people who actually know the languages.

**Comment by sauravrao637** [comment]
I agree with @Stypox  as if some translation is wrong one would notice and change but if it's not there it might not get there ever.

**Comment by mhmdanas** [comment]
@sauravrao637 Google Translate and friends can create confusing translations, which in my book are worse than no translation at all.

**Comment by sauravrao637** [comment]
> @sauravrao637 Google Translate and friends can create confusing translations, which in my book are worse than no translation at all.

okay I'll revert the last commit :) 
and open an issue to add translations in other languages.

**Review by mhmdanas — APPROVED** [review]

**Review by evrifaessa — APPROVED** [review]

**Comment by mhmdanas** [comment]
Thanks!

**Comment by opusforlife2** [comment]
@mhmdanas Question. There are 6 CI tests for this PR. Do all of them produce their own artifact APKs? Or is there a single artifact produced for the entire PR at its latest commit?

**Comment by sauravrao637** [comment]
@opusforlife2 this is meta improvement and doesn't affect the apk produced  so all artifacts are useless with this PR

**Pull request merged**